lib/l10n/app_en.arb

```
@@ -1273,8 +1273,13 @@
   "@tradesStatLabel": {"description": "Reputation stat label: trades"},
   "daysActiveStatLabel": "days active",
   "@daysActiveStatLabel": {"description": "Reputation stat label: days active"},
-  "timeToTakeOrder": "TIME TO TAKE THIS ORDER",
-  "@timeToTakeOrder": {"description": "Countdown label on the take order screen"},
+  "createdOnLabel": "Created on",
+  "@createdOnLabel": {"description": "Label above the order creation date"},
+  "timeRemainingLabel": "Time remaining: {time}",
```

RuboCop (1.89.0)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)